packages/focusmanager/src/react/use-focus-manager.ts

```diff
@@ -16,7 +16,7 @@ const FOCUSABLE_SELECTORS = [
 ]
 
 export default function useFocusManager(
-  ref: RefFor<'div'>,
+  ref: React.RefObject<HTMLDivElement>,
```

Comment:
In the future, steer clear of using the global React Object which is now an anti-pattern for current React settings which we will eventually move to. Once we do, we will have to update every instance in each package to use the named exports which will dramatically decrease the bundle size for our users. 😄 

This lib is about a 2 yrs behind in React conventions.

Reply:
Makes sense